testpilot/frontend/static-src/app/views/experiment-row-view.js

```diff
@@ -13,10 +16,6 @@ export default BaseView.extend({
                 </header>
                 <p data-hook="description"></p>
               </div>
-               <div class="experiment-actions">
```

Comment:
Hey, let's keep these in for now, actually, i don't want to back them out until we've replaced them with user counts